### PR TITLE
fix: set the js-ceramic image when scheduling k8 deploy

### PR DESCRIPTION
**Pull request by 3benbox**

### CODE REVIEW

**Review by nathanielc — APPROVED** [review]

File: ci-scripts/schedule_k8s_deploy.sh [hunk 1]
```diff
@@ -9,34 +9,46 @@ branch=${TEST_BRANCH-main}
...
+  "{                                                            \
+    \"id\":     {\"S\": \"$id\"},                               \
+    \"job\":    {\"S\": \"$job_id\"},                           \
+    \"ts\":     {\"N\": \"$now\"},                              \
+    \"ttl\":    {\"N\": \"$ttl\"},                              \
+    \"stage\":  {\"S\": \"queued\"},                            \
+    \"type\":   {\"S\": \"workflow\"},                          \
+    \"params\": {                                               \
+      \"M\": {                                                  \
+        \"name\":     {\"S\": \"Deploy CERAMIC ONE\"},          \
```
Comment (smrz2001) [inline ci-scripts/schedule_k8s_deploy.sh:39]: Can we update the string to something like `Deploy k8s dev CERAMIC ONE`? We don't currently display which env is being deployed to 😬
```suggestion
        \"name\":     {\"S\": \"Deploy k8s $network CERAMIC ONE\"},          \
```

Comment (smrz2001) [reply ci-scripts/schedule_k8s_deploy.sh:39]: You'll have to update the indentation of the `\`s...

File: ci-scripts/schedule_k8s_deploy.sh [hunk 2]
```diff
@@ -9,34 +9,46 @@ branch=${TEST_BRANCH-main}
...
+        \"org\":      {\"S\": \"3box\"},                        \
+        \"repo\":     {\"S\": \"ceramic-infra\"},               \
+        \"ref\":      {\"S\": \"$branch\"},                     \
+        \"workflow\": {\"S\": \"update_image.yml\"},            \
+        \"labels\":   {\"L\": [{\"S\": \"deploy\"}]},           \
+        \"inputs\":   {                                         \
+          \"M\": {                                              \
+            \"ceramic_one_image\": {\"S\": \"$image\"},         \
+            \"ceramic_image\":     {\"S\": \"$ceramic_image\"}, \
+            \"environment\": {\"S\": \"$environment\"}          \
```
Comment (smrz2001) [inline ci-scripts/schedule_k8s_deploy.sh:49]: minor: can you align the the `{` for `environment` with the braces above it?

File: ci-scripts/schedule_k8s_deploy.sh (hunk 1 above)
Comment (smrz2001) [reply ci-scripts/schedule_k8s_deploy.sh:39]: And feel free to setup the display string however is most clear as long as the network name is included.

**Review by smrz2001 — APPROVED** [review]
A couple of minor comments, otherwise LGTM!

File: ci-scripts/schedule_k8s_deploy.sh (hunk 2 above)
Comment (3benbox) [reply ci-scripts/schedule_k8s_deploy.sh]: updated!

**Pull request merged**